evergreen.yml

```diff
@@ -190,10 +196,10 @@ functions:
         aws_key: ${release_aws_key}
         aws_secret: ${release_aws_secret}
         local_file: mongo-powerbi-connector/MongoDBAtlasODBC-signed.pqx
-        remote_file: mongo-powerbi-connector/MongoDBAtlasODBC-${MONGO_POWERBI_CONNECTOR_VERSION}.pqx
+        remote_file: mongo-powerbi-connector/MongoDBAtlasODBC-${RELEASE_VERSION}.pqx
```

Comment:
Just to double-check.  Did you try running the `test-release` task?  If so, did it create the pqx file with the expected name: mongo-powerbi-connector/MongoDBAtlasODBC-0.0.0.pqx (or whatever the RELEASE_VERSION was)?

I was hitting some issues while working on the integration testing where the expansion wouldn't always occur when I had curly braces around the variable.  Which seems to sometimes expand the variable but not always (strange).  I then saw this in their [documentation](https://github.com/evergreen-ci/evergreen/wiki/Evergreen-Best-Practices#expansions).  I know we have this same syntax in other repos so if it is working then we can keep it as is.

Reply:
I can't see the task when I issue a patch direct to evergreen.